### PR TITLE
Removes cryogenic fluid from fire extinguishers because fuck you.

### LINKED ISSUE #7190: fuckn extinguishers no longer damage slime
In-Game report from glordqtinkle:
Server info: 27403 (Hippie Station)

>General description of the issue
WHATS UP WITH THAAAAAAAAAAAAAAAAT

>What you expected to happen
WHAT IS THE DEAL

>What actually happened
WITH

>Steps to reproduce if possible
THAAAAAAAAAAAAAAAAAAAAAAAAAT

### LINKED ISSUE #7347: Monkey cubes not expanding with fire extinguishers
Found during testing for my PR to fix fire extinguishers not damaging slimes correctly, monkey cubes do not create monkeys or react at all to water in any way. Probably a result of the TG Menace messing with the water code. This may also be happening to dehydrated carps.

The expand proc is working correctly as you can still activate the cube by forcing the proc with admin powa, so it's probably the water being fucked up or something.

### PR DESCRIPTION
**Pull request by praisenarsie**
Fixes #7347 
Fixes #7190 
Fixes #7355

you could've done this fix yourself john!!!

also what does cryogenic fluid actually do to the fires

### CODE REVIEW

**Comment by Jujumatic** [comment]
won't this stop fire extinguishers from... extinguishing fires. cryogenic fluid is what caused the fires to get extinguished, plus its super helpful for when the fire is out but you're still incinerating as it helps cool you down much faster

**Comment by praisenarsie** [comment]
extinguishers remove fire and half the temperature

also this appears to not be working but i got a feeling it's due to not having the relevant upstream already

**Comment by JohnGinnane** [comment]
@praisenarsie I could've but I wasn't sure if we wanted to remove this or not. Technically this is a bugfix but it's removing one of our features so I presume needs input

File: code/game/objects/items/extinguisher.dm [hunk 1]
```diff
@@ -43,8 +43,6 @@
 /obj/item/extinguisher/New()
 	..()
 	create_reagents(max_water)
```
Comment (JohnGinnane) [inline code/game/objects/items/extinguisher.dm:45]: I think you need to add water
```
reagents.add_reagent("water", max_water)
```

Comment (praisenarsie) [reply code/game/objects/items/extinguisher.dm:45]: thats what it is upstream but upstream might be a bit brogged so idk

**Comment by praisenarsie** [comment]
fuck you @chnkr

**Comment by Jujumatic** [comment]
wtf dont ever speak to my dad like that again

**Comment by ghost** [comment]
wtf my wife is your dad now?

**Comment by JohnGinnane** [comment]
image

this is literally the opposite of a sunflower issue because at the moment slimes are technically overpowered and monkey cubes are underpowered

I can merge this as a fix and we can look into adding cryo back in the future

@Carbonhell can we merge this for now so we have one less bug that kinda messes with people?

**Comment by JohnGinnane** [comment]
@praisenarsie `did you test this`

**Comment by praisenarsie** [comment]
it doesn't work but i'm about to fix that in a minute

**Comment by praisenarsie** [comment]
this SHOULD work since it's the same as /tg/ but i haven't tested it yet.

**Review by JohnGinnane — APPROVED** [review]

**Comment by earthmoonjune** [comment]
+1

**Comment by Jujumatic** [comment]
test it reeeeee

**Comment by praisenarsie** [comment]
Closed in favour of #7616 since I was too fucking lazy to test this and it would've took like 10 minutes aaaaaaa.

**Pull request closed without merging**